modules/pipelines-web-console.adoc

```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
 //
-// * layered-products-web-console.adc
+// * products-web-console.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="pipelines-web-console_{context}"]
 = {pipelines-title} in the web console
 
-{pipelines-title} is a cloud-native, continuous integration and continuous delivery (CI/CD) solution based on Kubernetes resources. You can use the OperatorHub in the {product-title} web console to install the {pipelines-title} Operator to create and modify pipeline objects.
```

Comment:
Does the style guide permit "allows you"?

Otherwise LGTM.

Reply:
I think it might be one that is discouraged. I'll revise to "you can"

Reply:
Updated!